px-context-browser.html

```diff
@@ -134,7 +135,7 @@
       <!-- NON-EMPTY FAVORITES STATE -->
       <px-context-browser-panel id="favoritedPanel" slot="dropdown-content" collapsed$="[[collapsed]]" show-loading-bar="[[favoritedSyncing]]">
         <template is="dom-if" if="[[!_showFavoritedEmptyState(favorited, favorited.*)]]">
-          <px-context-browser-filter slot="filter" active$="[[showFilter]]" filter="{{favoritedFilter}}" filtering="{{filtering}}"></px-context-browser-filter>
+          <px-context-browser-filter slot="filter" active$="[[showFilter]]" filter="{{favoritedFilter}}" filtering="{{filtering}}" language="{{language}}" use-key-if-missing="{{useKeyIfMissing}}" resources="{{resources}}"></px-context-browser-filter>
```

Comment:
See comment in px-context-browser-filter.html, if we do that we can remove these properties and pass in placeholder.